dcos/dcos-ui
# DCOS-47175: Update Table component in nodes page

Use the new version of the ui-kit to enable the nodes table
to auto resize columns based on their content width.

Closes https://jira.mesosphere.com/browse/DCOS-47175

~Currently blocked by https://github.com/dcos-labs/ui-kit/pull/276~

## Testing
1. Go to Nodes tab
2. Verify that the table doesn't jump as the data is loaded
3. Verify that the table works and the columns are as wide as the content

## Trade-offs
~I don't think that the new table looks better. I considered adding some minimal width to the columns, but then we would have the old table in almost every case (as the health column for example will never be wider than the old value of 120px). @lsamimi , @mperrotti , what do you think?~

I added a loader if the data array is empty, but some of the values in the table are slower to load, which causes it to move.

Also fixed some unrelated failing snapshots and some other minor things.

## Dependencies
~https://github.com/dcos-labs/ui-kit/pull/276~

## Screenshots

### Before
![screenshot from 2019-01-16 11-01-56](https://user-images.githubusercontent.com/40791275/51237982-6c983980-197e-11e9-836c-bc83e98f7af9.png)

### After (old)
![screenshot from 2019-01-16 11-04-40](https://user-images.githubusercontent.com/40791275/51238045-92bdd980-197e-11e9-8fc9-95f9c8861d72.png)

### After (new)
![screenshot from 2019-01-17 09-51-21](https://user-images.githubusercontent.com/40791275/51303824-31117400-1a3f-11e9-8977-7aae9e959471.png)
